pkg/controller/core.oam.dev/v1alpha2/application/apply_test.go

```diff
@@ -132,81 +123,6 @@ var _ = Describe("Test Application apply", func() {
 		Expect(k8sClient.Delete(context.TODO(), &ns)).Should(Succeed())
 	})
 
-	It("Test update or create component", func() {
```

Comment:
So this can be refactored to test component level revision

Reply:
I think it's ok to use controllerRevision

Reply:
got it, will add new test for new component revision mechanism